lessons/Data-Science/measures-of-center/langs/en-us/index.adoc

```diff
@@ -15,8 +15,9 @@
 | Students will be able to...
 
 @objectives
-@objective{measures-of-center-pyret}
-@objective{Recognize that the Measures of Center we can compute are not all necessarily typical of the distribution.}
+@objective{measures-of-center-proglang}
+@objective{histograms-and-measures-of-center}
+@objective{best-measure-of-center}
```

Comment:
These `@objective`s should be moved into the lesson sections where they are addressed. That makes the lesson more readable for authors ("oh, this section 
 is intended to address X!"), and also easier to dissect if we ever want to split a section out into a new lesson (the objective will move with it, instead of being orphaned)